qiskit_metal/_defaults.py

```diff
@@ -127,7 +127,7 @@ def update_default_options(self,
                                cust_value: Dict = None):
         """Allow instance of class to update the default_options.
 
-        Args:
+        Arguments:
```

Comment:
Hi @grace-harper-ibm ,
I am somewhat confused on this change here. This is inconsistent with the VScode default, which is `Args:`. Making this `Arguments:` will constantly cause inconsistency in future PR that use the default. Any particular reason you went ahead with this change (in this PR)?

Reply:
`Args:` and `Arguments:` are both acceptable from a docs standpoint, Presently, they are used interchangeably throughout the code.

Reply:
That having been said, changing `Args:` to `Arguments` in this case is unnecessary.

Reply:
I agree they are both acceptable from a docstring stand point, but given that out IDE of choice uses `Args:` by default, changing things the opposite way feels like asking for trouble.

Reply:
Hey Marco
That's a good point. I just saw things were inconsistent and so I arbitrarily chose one to standardize across the board
Let me remedy it